packages/ag-charts-community/src/util/interpolating.ts

```diff
@@ -0,0 +1,6 @@
+// FIXME: Use symbol (requires TS >=4.9)
+export const interpolate = '$interpolate';
+
+export interface Interpolating<T = any> {
+    $interpolate: (other: T, d: number) => T;
```

Comment:
Why use a special symbol for this case, wouldn't a standard interface be sufficient?

Reply:
I was going to make it a symbol, but the TS version we’re on doesn’t support it